### PR TITLE
Update Github actions

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
This PR:
- Update debug workflow
- Add release workflow

I wait you add keystore and API KEY on Github secrets to doing changes.

After we submit 3.0 version on Play Store, i update release workflow to create release on Github (with APK and changelog).

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -18,7 +18,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```
Comment (pnu-s) [inline .github/workflows/main.yml:21]: **question**: Why do we change to v3 and what does it change?

(same question for `upload-artifact`)

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -18,7 +18,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```
Comment (pnu-s) [inline .github/workflows/main.yml:25]: **question**: Why do we need to add this part?

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+name: Release
+env:
+    EXODUS_API_KEY: ${{secrets.EXODUS_API_KEY}}
+on:
+  push:
+    tags:
+      - release-v*
+
```
Comment (pnu-s) [inline .github/workflows/release.yml:9]:
```suggestion
```

**suggestion**: Unnecessary line

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  push:
+    tags:
+      - release-v*
+      
+  workflow_dispatch:
+jobs:
+    release:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Checkout the code ✅
```
Comment (pnu-s) [inline .github/workflows/release.yml:15]: **suggestion**: If we want to add icons, can we put it in the main workflow as well?

**Review by pnu-s — CHANGES_REQUESTED** [review]
Thanks @Jean-BaptisteC ! :+1: 

I added a couple of comments, I'll let you take a look at them

File: .github/workflows/main.yml (hunk 1 above)
Comment (Jean-BaptisteC) [reply .github/workflows/main.yml:21]: Checkout and upload-artifact 3.0 update node js to [16.0](https://github.com/nodejs/node/releases/tag/v16.0.0).
This version of node js including performance tweaks and improvements.

File: .github/workflows/main.yml (hunk 2 above)
Comment (Jean-BaptisteC) [reply .github/workflows/main.yml:25]: I add this part to be sure WKF don't use latest version of Java.

**Comment by Jean-BaptisteC** [comment]
@codeurimpulsif  Add Github secrets:

- [ ] storePassword
- [ ] keyAlias
- [ ] keyPassword
- [Source](https://proandroiddev.com/how-to-securely-build-and-sign-your-android-app-with-github-actions-ad5323452ce)
- [ ] Keystore
- [ ] Keystore_PassPhrase
- [Convert keystore to GPG](https://stefma.medium.com/how-to-store-a-android-keystore-safely-on-github-actions-f0cef9413784)

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          run: ./gradlew assemble
+        - name: Execute exodus-standalone 🔎
+          uses: docker://exodusprivacy/exodus-standalone:v1.3.1
+          with:
+            args: /github/workspace/app/build/outputs/apk/release/app-release-signed.apk
+        - name: Upload APK 🚀
+          uses: actions/upload-artifact@v3
+          with:
+            name: app_release
+            path: app/build/outputs/apk/release/*.apk
```
Comment (pnu-s) [inline .github/workflows/release.yml:30]: **suggestion**: If we know the exact path of the apk, can we put it instead of a wildcard?

**Review by pnu-s — APPROVED** [review]

**Comment by codeurimpulsif** [comment]
I just add public and private keys PEM to secrets (because we can't add a keystore file in secrets).

**Comment by Jean-BaptisteC** [comment]
@theimpulson can you add management keystore on [release config](https://github.com/Exodus-Privacy/exodus-android-app/blob/kotlinRewrite/app/build.gradle)

**Comment by theimpulson** [comment]
> @theimpulson can you add management keystore on [release config](https://github.com/Exodus-Privacy/exodus-android-app/blob/kotlinRewrite/app/build.gradle)

Done at https://github.com/Exodus-Privacy/exodus-android-app/commit/c374e7c54e10df7168e4028b2ef39fff6c4257c9

**Comment by Jean-BaptisteC** [comment]
@pnu-s Can you add API KEY on Secrets ?

**Comment by pnu-s** [comment]
Secret is already available, name is `EXODUS_API_KEY`

**Pull request merged**